src/apps/explorer/components/SummaryCardsWidget/summaryData.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Same for this file, is it still needed? I might have missed but on a first glance did not find it being used anywhere

Reply:
This can probably be removed as it's derived data and not what the subgraph is returning